components/benefit_sponsors/app/models/benefit_sponsors/benefit_sponsorships/benefit_sponsorship_director.rb

```diff
@@ -12,6 +12,7 @@ def process(benefit_sponsorships, event)
       business_policy_name = policy_name(event)
 
       benefit_sponsorships.no_timeout.each do |benefit_sponsorship|
+        @logger.info "Event (#{event}) process started for Employer #{benefit_sponsorship.legal_name}(#{benefit_sponsorship.fein})"
         begin
           business_policy = business_policy_for(benefit_sponsorship, business_policy_name)
           sponsor_service_for(benefit_sponsorship).execute(benefit_sponsorship, event, business_policy)
```

Comment:
what do you think about adding a logging after it finished renewing an employer.
```
@logger.info "Event (#{event}) process finished for Employer #{benefit_sponsorship.legal_name}(#{benefit_sponsorship.fein})"
```